src/tests/notify.bats

```diff
@@ -36,7 +36,7 @@ setup() {
     SLACK_PARAM_CUSTOM=$(cat $BATS_TEST_DIRNAME/sampleCustomTemplateWithLink.json)
     SLACK_DEFAULT_CHANNEL="xyz"
     BuildMessageBody
-    EXPECTED=$(echo "{ \"blocks\": [ { \"type\": \"section\", \"text\": { \"type\": \"mrkdwn\", \"text\": \"Sample link using environment variable in markdown <${TESTLINKURL}|LINK >\" } } ], \"text\": \"\", \"channel\": \"$SLACK_DEFAULT_CHANNEL\" }" | jq)
+    EXPECTED=$(echo "{ \"blocks\": [ { \"type\": \"section\", \"text\": { \"type\": \"mrkdwn\", \"text\": \"Sample link using environment variable in markdown <${TESTLINKURL}|LINK >\" } } ], \"text\": \"\", \"channel\": \"$SLACK_PARAM_CHANNEL\" }" | jq)
```

Comment:
I do not believe these should be changed in the test. The default channel value is what is mocked above. What was the reason for the name change here?

Reply:
I guess I was confused why it's using `SLACK_DEFAULT_CHANNEL` instead of `SLACK_PARAM_CHANNEL`.

https://github.com/CircleCI-Public/slack-orb/pull/259/files#diff-ef6c7403db52e40f6117d31e16d05d41ad923c6f2673e0a04cbcf97c62433c27L113-R116

Why is `SLACK_PARAM_CHANNEL` checked, but the message mentions `SLACK_DEFAULT_CHANNEL`?

Reply:
The value is inferred. If you do not set the `channel` parameter, the value `$SLACK_DEFAULT_CHANNEL` will be used. 
https://github.com/CircleCI-Public/slack-orb/blob/4bf71a8c2fe5666d3adc52229ce9ab030464bab1/src/commands/notify.yml#L72

This means that `SLACK_PARAM_CHANNEL` will contain the string literal `$SLACK_DEFAULT_CHANNEL`, this is evaluated so that either a string OR an environmentvariable could be used.

When we change the default of $SLACK_DEFAULT_CHANNEL we test the entire scope of this process of supporting default channels via contexts.

Reply:
I see. Ok, I've reverted it.